<?xml version="1.0" ?>
<!-- =================================================================================== -->
<!-- |    This document was autogenerated by xacro from plug_handle_two.xacro          | -->
<!-- |    EDITING THIS FILE BY HAND IS NOT RECOMMENDED                                 | -->
<!-- =================================================================================== -->
<robot name="plug_handle" xmlns:xacro="http://www.ros.org/wiki/xacro">
  <link name="plug_link">
    <visual>
      <!--${-M_PI/2} ${-M_PI} ${-M_PI/2}-->
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="package://objects/meshes/plug/plug.dae"/>
      </geometry>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="package://objects/meshes/plug/plug.dae"/>
      </geometry>
    </collision>
  </link>
  <!--joint name="link_c_p" type="fixed">
	<origin xyz="0 0 0" rpy="0 0 0"/>
	<parent link="plug_link"/>
	<child  link="link_cylinder"/>
</joint>


<link name="link_cylinder">
	<visual>
		<origin xyz="${-1*dx} 0 ${-1*dz}" rpy="0 ${-M_PI/2} 0"/>
		<geometry>
			<cylinder length="${c_l}" radius="${c_r}"/>
		</geometry>
		<material name="grey">
  		         <color rgba="0.6 0.6 0.6 1"/>
  	       </material>
        </visual>

</link-->
</robot>

